### PR TITLE
add test 2sct polaris router

### PR DESCRIPTION
**Pull request by dongyinuo**
## PR Type
add sct-stater-polaris-router juint

image

## Describe what this PR does for and how you did.
add sct-stater-polaris-router juint

## Adding the issue link (#xxx) if possible.
https://github.com/Tencent/spring-cloud-tencent/issues/682

## Checklist

- [ ] Coverage from `Codecov Report` should not decrease (which means adding junit tests for adding or updating code).
- [ ] All junit tests passing.
- [ ] Add copyright holder at the beginning of .java file if it is new.
- [ ] Add information of this PR to CHANGELOG.md in root of project.

## Checklist (Optional)

- [ ] Will Pull Request to branch of 2020.0 and hoxton.
- [ ] Add documentation in javadoc or comment below the PR if necessary.

### CODE REVIEW

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisMetadataRouterPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	@Test
+	public void setEnabled() {
+		properties.setEnabled(false);
+		assertThat(properties.isEnabled()).isEqualTo(false);
+	}
+
+	@Test
+	public void testToString() {
+		System.out.println(properties);
```
Comment (SkyeBeFreeman) [inline spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisMetadataRouterPropertiesTest.java]: 可以改为设置值，然后将tostring的调用结果进行对比。

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisNearByRouterPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+	@Test
+	public void setEnabled() {
+		properties.setEnabled(false);
+		assertThat(properties.isEnabled()).isEqualTo(false);
+	}
+
+	@Test
+	public void testToString() {
+		System.out.println(properties);
```
Comment (SkyeBeFreeman) [inline spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisNearByRouterPropertiesTest.java]: 可以改为设置值，然后将tostring的调用结果进行对比。

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisRuleBasedRouterPropertiesTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+	@Test
+	public void setEnabled() {
+		properties.setEnabled(false);
+		assertThat(properties.isEnabled()).isEqualTo(false);
+	}
+
+	@Test
+	public void testToString() {
+		System.out.println(properties);
```
Comment (SkyeBeFreeman) [inline spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/config/properties/PolarisRuleBasedRouterPropertiesTest.java]: 可以改为设置值，然后将tostring的调用结果进行对比。

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/resttemplate/PolarisLoadBalancerRequestTests.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		assertThat(apply.getServiceId()).isEqualTo(calleeService);
+		assertThat(polarisLoadBalancerRequest.getRequest()).isEqualTo(request);
+		assertThat(polarisLoadBalancerRequest.getDelegate()).isEqualTo(mockLoadBalancerRequest);
+	}
+
+	static class MockLoadBalancerRequest implements LoadBalancerRequest {
+
+		@Override
+		public Object apply(ServiceInstance instance) throws Exception {
+			System.out.println("Mock LoadBalancerRequest");
```
Comment (SkyeBeFreeman) [inline spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/resttemplate/PolarisLoadBalancerRequestTests.java]: 不使用system.out输出，可以利用verify方法来判断调用次数

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]

File: spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/resttemplate/PolarisLoadBalancerRequestTests.java (hunk 4 above)
Comment (dongyinuo) [reply spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/resttemplate/PolarisLoadBalancerRequestTests.java]: 好的， 我统一该一版

**Comment by dongyinuo** [comment]
已统一修复

**Comment by SkyeBeFreeman** [comment]
@dongyinuo 查看一下单测问题

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/spring-cloud-tencent?pullRequest=731) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/spring-cloud-tencent?pullRequest=731) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: SkyeBeFreeman<br/>:x: dongyinuo<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/spring-cloud-tencent?pullRequest=731) it.</sub>

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]
@dongyinuo 修复单测错误

**Review by SkyeBeFreeman — DISMISSED** [review]

**Review by SkyeBeFreeman — APPROVED** [review]

**Pull request merged**